f2/src/utils/formik/validationSchema.js

```diff
@@ -0,0 +1,27 @@
+const Yup = require('yup')
+
```

Comment:
As we discussed, let move all the regex or Yup schema object into a separate package and export them.
Then we can import required regex or Yup schema object here.

Reply:
Regex is defined in a separate module, Yup schema is defined in the form now.